app/views/assessments/_handin_form.html.erb

```diff
@@ -69,7 +69,17 @@
             <% end %>
             
             <%= render partial: "submission_panel", locals: {repos: @repos, f: f}%>
-
+            <% filename_list = @assessment.handin_filename.split(".") %>
+            <% if (filename_list.size == 2) %>
+                <!-- file needs to have a period, else we ignore (e.g. for README files) -->
+                <div class="filename-check" id="filename-check">
+                  <div>
+                    Handin format:
+                    .<b id="handin-file-type"><%= filename_list[1] %></b> files
```

Comment:
Concatenate remainder of `filename_list`

Reply:
done